### PR TITLE
Support proto encoding for DASH

### PR DESCRIPTION
**Pull request by ganglyu**
#### Why I did it
We will use proto encoding for all DASH tables.

#### How I did it
Update gnmi server and client to support proto encoding.

#### How to verify it
Run unit test.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

### CODE REVIEW

File: test/test_gnmi_appldb.py [hunk 1]
```diff
@@ -5,410 +5,61 @@
...
-    def test_gnmi_update_normal_01(self, test_data):
-        clear_appl_db('DASH_QOS')
-        clear_appl_db('DASH_VNET')
+    def test_gnmi_update_normal_01(self):
+        proto_bytes = b"\n\x010\x12$b6d54023-5d24-47de-ae94-8afe693dd1fc\x1a\x17\n\x12\x12\x10\r\xc0-\xdd\x82\xa3\x88;\x0fP\x84<\xaakc\x16\x10\x80\x01\x1a\x17\n\x12\x12\x10-\x0e\xf2\x7f\n~c_\xd8\xb7\x10\x84\x81\xd6'|\x10\x80\x01\x1a\x17\n\x12\x12\x10\x1bV\x89\xc8JW\x06\xfb\xad\b*fN\x9e(\x17\x10\x80\x01\x1a\x17\n\x12\x12\x107\xf9\xbc\xc0\x8d!s\xccVT\x88\x00\xf8\x9c\xce\x90\x10\x80\x01\x1a\x17\n\x12\x12\x10\tEb\x11Mf]\x12\x17x\x99\x80\xea\xd1u\xb4\x10\x80\x01\x1a\x17\n\x12\x12\x10\x1f\xd3\x1c\x89\x99\x16\xe7\x18\x91^0\x81\xb1\x04\x8c\x1e\x10\x80\x01\x1a\x17\n\x12\x12\x10\x06\x9e55\xdb\xb5&\x93\x99\xfaC\x81\x16P\xdc\x1d\x10\x80\x01\x1a\x17\n\x12\x12\x10&]U\x96e4\xf4\xd2'&\x04i\xdf\x8dA\x9f\x10\x80\x01\x1a\x17\n\x12\x12\x108\xd5\xa3*\xe7\x80\xdc\x1e\x80f\x94\xb7\xb6\x86~\xcd\x10\x80\x01\x1a\x17\n\x12\x12\x101\xf0@F\nu+}\x1e\"\\\\\xdb\x01\xe3\x82\x10\x80\x01\"\x05vnet1\"\x05vnet2\"\x05vnet1\"\x05vnet2\"\x05vnet2\"\x05vnet1\"\x05vnet2\"\x05vnet2\"\x05vnet1\"\x05vnet1"
```
Comment (liuh-80) [inline test/test_gnmi_appldb.py:335]: Is there are util to convert json format data to protocol buffer format?

File: sonic_data_client/mixed_db_client.go [hunk 2]
```diff
@@ -571,98 +582,28 @@ func (c *MixedDbClient) handleTableData(tblPaths []tablePath) error {
...
 
-			for _, dbkey := range dbkeys {
-				tableKey := strings.TrimPrefix(dbkey, tblPath.tableName + tblPath.delimitor)
-				err = c.DbDelTable(tblPath.tableName, tableKey)
-				if err != nil {
-					log.V(2).Infof("swsscommon delete failed for  %v, dbkey %s", tblPath, dbkey)
-					return err
-				}
+		if tblPath.operation == opRemove {
+			err = c.DbDelTable(tblPath.tableName, tblPath.tableKey)
```
Comment (liuh-80) [inline sonic_data_client/mixed_db_client.go]: The delete all keys of a table code seems been removed?

File: test/test_gnmi_appldb.py (hunk 1 above)
Comment (ganglyu) [reply test/test_gnmi_appldb.py:335]: We will not provide json format, and we need Ze to generate protobuf builder

File: sonic_data_client/mixed_db_client.go (hunk 2 above)
Comment (ganglyu) [reply sonic_data_client/mixed_db_client.go]: yes

Comment (liuh-80) [reply sonic_data_client/mixed_db_client.go]: Thanks, client still can delete multiple keys with 1 gnmi API call: add multiple delete operation in 1 APi call, every delete operation will delete 1 key.

**Pull request merged**